js/common.d.ts

```diff
@@ -262,6 +262,13 @@ export type ValidationMessageMode = 'always' | 'auto';
  */
 export type ValidationStatus = 'valid' | 'invalid' | 'pending';
 
+/**
+ * @public
+ * @namespace DevExpress.common
+ */
+
+export type ValueChangeMode = 'instant' | 'eventual';
```

Comment:
```suggestion
export type SliderValueChangeMode = 'instant' | 'eventual';
```

Actual name seems to be too common